packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -95,6 +95,25 @@ pub struct ExperimentConfig {
     clap(global = true, short = 'w', long, default_value_t = num_cpus::get(), validator = at_least_one, env = "HASH_WORKERS")
     )]
     pub num_workers: usize,
+
+    #[cfg_attr(feature = "clap", clap(global = true, long, default_value_t = 0))]
+    /// Initial size of the V8 heap.
+    ///
+    /// Setting this value allows to avoid garbage collection while the heap is small enough.
+    pub v8_initial_heap_constraint: usize,
```

Comment:
Can we not directly mention v8 but refer to the JavaScript runner instead?

Reply:
Yes but this is very low level. I would vote to keep the V8 variable name somewhere, at least in the doc, to make it easier to find for users.

Reply:
Why does "V8" matters to the users at all? Also, it's confusing: "I have 10 runners, if I set the constraint to 10 GB, do I need 100 GB?" For the user, it's unimportant, what internal API is used. If they are facing OOM errors because the JS runner crashed, they want to increase the limit for the JS runner, not for v8.

Reply:
Why does "V8" matters to the users at all? Also, it's confusing: "I have 10 runners, if I set the constraint to 10 GB, do I need 100 GB?" For the user, it's unimportant, what internal API is used. If they are facing OOM errors because the JS runner crashed, they want to increase the limit for the JS runner, not for v8.

Reply:
Ok that makes sense.

Reply:
Removed mentions of V8 in https://github.com/hashintel/hash/pull/446/commits/ab01617505d8a125a66d634620fcc2c5b5c074c8

Reply:
Great!